.coveragerc

```diff
@@ -0,0 +1,3 @@
+[run]
+omit = **/migrations/*,**/settings/*,**/wsgi.py,**/tests/*,**/manage.py
```

Comment:
We probably want to omit `src/archivematicaCommon/lib/externals/` (but not `src/dashboard/src/external/archivematica-fpr-admin/` though it may be irrelevant since the fpr-admin will be installed as a proper dependency once the translations work #506 comes in)